packages/product-components/src/components/TokenIconSet/TokenIconSet.tsx

```diff
@@ -0,0 +1,64 @@
+import { AssetLogo, useElevation } from '@trezor/components';
+import { getCoingeckoId } from '@suite-common/wallet-config';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { TokenInfo } from '@trezor/connect';
+import { getContractAddressForNetwork } from '@suite-common/wallet-utils';
+
+import styled, { css } from 'styled-components';
+import { borders, Elevation, mapElevationToBackground, mapElevationToBorder } from '@trezor/theme';
+
+export type TokenIconSetProps = {
+    network: NetworkSymbol;
+    tokens: TokenInfo[];
+};
+
+const IconContainer = styled.div<{ $length: number }>`
+    width: 26px;
```

Comment:
<img width="1030" alt="Screenshot 2024-10-11 at 12 22 54" src="https://github.com/user-attachments/assets/04d6bf4d-e13c-4dd1-9759-01b9842b3f9e">
Tokens text is moved

Reply:
fixed this